src/content/docs/more-integrations/best-practices-integration.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Best practices for integration users
+tags:
+  - Best practices
+  - Integrations
+metaDescription: Here are some best practices for controlling how you expose your New Relic data when you're using an integration.
+freshnessValidatedDate: 2024-03-25
+---
+
+As a New Relic administrator, you must operate your New Relic organization with the principle of least privilege. By following this principle, you can reduce the risk of unauthorized access to sensitive data. This includes assigning permissions based on job roles and responsibilities. In a [New Relic organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/), account-based access allows you to control what information is available to the users in your organization. This ensures that sensitive data remains secure and only authorized personnel can view it. 
+
+Dealing with integrations with third-party systems can complicate application access. For example, users in the other system may not have access to New Relic. The system may also lack centralized permissions or controls. Learn about best practices for data sharing on this page. Take control of how you want to expose your New Relic data during integrations.
+
+## Account and API key scope [#account-api-key-scope]
+
+The first thing to consider when controlling access to your New Relic data is the scope of your account. Your organization can have multiple accounts and users are then given access to accounts. This allows you to limit the data these users can see. If your organization only has a single account, this means that all users have access to the same information. In this situation, you may only want to expose some data through an integration.
+
+Some integrations with New Relic take advantage of the ability to generate an API key to expose your data to other systems (for example, the Atlassian integration). If a user has access to 3 out of 5 accounts in a New Relic organization, their generated API key will have access to those same 3 accounts. 
+
+<Callout variant="important">
+Note that an API key inherits the account permissions of the user who generated it.
+</Callout>
+
+It's important to understand this distinction when setting up an integration. For example, you may want to make sure about generating an API key from a user who has the correct permissions. You may want the user to only have access to certain accounts in the application. This limits what they can see and share with others. For example, say you have more users in your Atlassian application than in your New Relic accounts, and you configure the Atlassian integration. The users in the Atlassian integration will be able to see the New Relic data shared with Atlassian in the Atlassian application. Regardless of whether those users have access to the corresponding New Relic accounts.
+
+
+## Integration user leverage [#leverag-integration-user]
```

Comment:
This title is likely accurate but it's a little confusing at first glance, anyway to clarify it?